src/components/Taskbar.tsx

```
@@ -0,0 +1,85 @@
+            <>
+              {openWindows.map((win) => (
+                <button
+                  key={win.id}
+                  onClick={() => onWindowClick(win.id)}
+                  className={`relative flex items-center justify-center w-10 h-10 rounded-xl bg-[var(--bg-elevated)] border transition transform hover:scale-110 ${
+                    win.isMinimized
+                      ? "border-[var(--border-subtle)] opacity-60"
+                      : "border-[var(--coral-bright)] shadow-[0_0_12px_var(--shadow-coral-mid)]"
+                  }`}
+                  title={win.title}
+                >
```

Biome (2.4.6)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)